### ISSUE TITLE
Invalid Status: 401 communication error

### ISSUE THREAD

**Issue by adamskoog** [nzbget/nzbget#641]
I'm getting this error message in Edge. Glad for the above to know that I can use Chrome to get around it. Nzbget is running on Windows 10, I have a Linux VM running my web proxy (nginx) with a certificate from Let's Encrypt. I get this error whether trying to view it via https or http. I just updated to version 21.0 and tested again, still seeing this issue.

Using Edge and viewing the dev tools I see an error in the console:
> HTTP401: DENIED - The requested resource requires user authentication.
(XHR)POST - https://myserver.no-ip.org/nzbget/jsonrpc

This issue appears to be related: #476

**Comment by hugbug** [comment]
I'm not sure it is worth to trying to fix the issue with Edge considering how little users use this browser and the fact MS is going to ditch it anyway.

**Comment by adamskoog** [comment]
Yeah, seems that Edge is going to be using Chromium under the hood quite soon so it's really probably not something worth your time at this point. Feel free to close if no one has seen this on any other supported browser.

**Comment by mikenerone** [comment]
If it helps anybody: I recently experienced this error message with Chrome running on both a Windows PC and a Macbook. I was able to resolve it by removing cookies. Note that the EditThisCookie extension wasn't sufficient for some reason. I had to click the small warning icon in the URL bar (triangle with an exclamation mark in it), then "Cookies" to use Chrome's built-in cookie management window.
1. In the "Allowed" tab (where it says "The following cookies were set when you viewed this page"), remove all listed cookies.
2. Make sure the "Blocked" tab is empty.
